mmocr/models/textdet/detectors/base_detector.py

```diff
@@ -3,40 +3,19 @@
 
 import mmcv
 
-from mmocr.core import imshow_pred_boundary, seg2boundary
+from mmocr.core import imshow_pred_boundary
 
 
-class TextDetectorMixin:
-    """The class for implementing text detector auxiliary methods."""
+class BaseDetector:
```

Comment:
BaseTextDetector?

Reply:
I think use Mixin(TextDetectorMixin/HeadMixin) for three reasons:

1. OCRMaskRCNN(BaseTextDetector, MaskRCNN)/FCENet(BaseTextDetector, SingleStageTextDetector) which are multiple inheritance，　usually composing a BasicClass and other Interfaces(for union API). TextDetectorMixin/HeadMixin has the only effect: loss and process, which is for all detectors, so it should be an Interface.
2. BaseDetector has the same name as mmdet(mmdet/model/base, which is real base detector)
3. The post-process will be refactored in mmlab2.0 which mean xxxMixin may be deleted in the further, so it may be better for keeping the same name to avoid frequently changing